src/utils.ts

```diff
@@ -101,6 +104,10 @@ export const SETTINGS_DEFAULT_ATTACH_INTERVAL = 200;
 const WIN_APP_DATA = process.env.LOCALAPPDATA || '/';
 const msEdgeBrowserMapping: Map<BrowserFlavor, IBrowserPath> = new Map<BrowserFlavor, IBrowserPath>();
 
+/** Build-specified flags. */
```

Comment:
hmmm, these are currently not playing nice with our tests. I'll see if I can find a fix